client/src/components/EmailAddressTable.tsx

```diff
@@ -1,34 +1,177 @@
-import _get from "lodash/get"
+import { Icon } from "@blueprintjs/core"
+import { IconNames } from "@blueprintjs/icons"
+import RemoveButton from "components/RemoveButton"
+import { MERGE_SIDES, setAMergedField } from "mergeUtils"
 import React from "react"
-import { Table } from "react-bootstrap"
+import { Button, Table } from "react-bootstrap"
 import utils from "utils"
 
 interface EmailAddressTableProps {
   label: string
   emailAddresses?: any[]
+  mergeMode?: boolean
```

Comment:
Instead of changing this component, I suggest to create a new, a separate component specific for the email address merge (e.g. `client/src/pages/admin/merge/MergeEmailAddresses.tsx`).